Microsoft.Toolkit.Uwp.UI.Controls.Primitives/Microsoft.Toolkit.Uwp.UI.Controls.Primitives.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+	<PropertyGroup>
+		<TargetFrameworks>uap10.0.17763</TargetFrameworks>
+		<Title>Windows Community Toolkit Controls</Title>
+		<Description>
+			This library provides XAML templated controls. It is part of the Windows Community Toolkit.
+
+			Controls:
+			- AdaptiveGridView: Presents items in an evenly-spaced set of columns to fill the total available space.
+			- DockPanel: Define areas where you can arrange child elements either horizontally or vertically, relative to each other.
+			- StaggeredPanel: Layout of items in a column approach where an item will be added to whichever column has used the least amount of space.
+			- UniformGrid: Presents items in an evenly-spaced set of rows or columns to fill the total available display space.
+			- WrapPanel: Positions child elements in sequential position from left to right and breaks content to the next line.
+		</Description>
+		<PackageTags>UWP Toolkit Windows Controls XAML WrapPanel Adaptive DockPanel StaggeredPanel Staggered UniformGrid Uniform Grid</PackageTags>
+		<!-- ARM64 builds for managed apps use .NET Native. We can't use the Reflection Provider for that. -->
+		<EnableTypeInfoReflection Condition="'$(Configuration)' == 'Debug'">false</EnableTypeInfoReflection>
+		<LangVersion>8.0</LangVersion>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
Are we using `ValueTuple` anywhere here still?

Reply:
Done 1e34abf